hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java

```diff
@@ -75,17 +73,15 @@
       defaultValue = "", tags = {ConfigTag.BALANCER}, description = "The " +
       "maximum size that can enter a target datanode in each " +
       "iteration while balancing. This is the sum of data from multiple " +
-      "sources. The default value is greater than the configured" +
-      " (or default) ozone.scm.container.size by 1GB.")
-  private long maxSizeEnteringTarget;
+      "sources. by default, we do not limit this value")
+  private long maxSizeEnteringTarget = Long.MAX_VALUE;
```

Comment:
Let's revert the configuration changes here and do it in a separate PR.

Reply:
sure , will do this!